src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE if exists hearing_outcome_aud ADD COLUMN fk_hearing_defendant_id integer ;
+
+END;
```

Comment:
ALTER TABLE hearing_outcome_aud
    ADD COLUMN IF NOT EXISTS id;
    
    ALTER TABLE hearing_outcome_aud 
    ADD CONSTRAINT fk_hearing_defendant_id FOREIGN KEY (id) REFERENCES {something?};

Reply:
If that column already exists ID, then ignore the first statement

Reply:
This is assuming the column name is 'id'